code/game/objects/items/weapons/grenades/clusterbuster.dm

```diff
@@ -169,11 +169,6 @@
 	desc = "No matter what, do not EVER use this."
 	payload = /obj/singularity
 
-/obj/item/grenade/clusterbuster/tools
```

Comment:
these do different things, can you bring it back under a different path?

Reply:
sure